crt_portal/cts_forms/views.py

```diff
@@ -96,7 +96,8 @@ def get_context_data(self, form, **kwargs):
 
         if current_step_name == 'Details':
             context.update({
-                'page_note': 'Continued'
+                'page_note': 'Continued',
+                'form_novalidate': True,
```

Comment:
I use this as well in my location PR, is it worth just setting it globally at this point? I think every form we have with validation is going to use it?

Reply:
That's a great point- I can make that tweak